src/tests/ftest/io/nvme_fragmentation.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2019-2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+
+import time
+from ior_test_base import IorTestBase
+
+class NvmeFragmentation(IorTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test class Description: Verify the drive fragmentation does free
+    the space and do not lead to ENOM_SPACE.
+
+    :avocado: recursive
+    """
+
+    def test_nvme_fragmentation(self):
+        """Jira ID: DAOS-2332.
+
+        Test Description:
+            Purpose of this test is to verify there is no Fragmentation
+            after doing some IO write/delete operation for ~hour.
+
+        Use case:
+        Create object with different transfer size
+        wait for Aggregation to happen
+        read the data again
+        Delete all the pool
```

Comment:
Deleting pool will just simply delete the pmemobj file, I suggest to delete all containers, so that huge amout of vea_free() will be called to deallocate NVMe blocks, then we'll be able to see how vea_free() performs on a fragmented pool.